internal/v1/mocks/compliance.go

```
@@ -77,6 +77,20 @@ func tailorings(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func tailoringsNotFound(w http.ResponseWriter, r *http.Request) {
+	if tutils.AuthString0 != r.Header.Get("x-rh-identity") {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	_, err := w.Write([]byte("{}"))
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
```

You can ignore both number of bytes written and error because response writer is an in-memory write.

---

Changed to the error being ignored as well